ucsmsdk/ucsxmlcodec.py

```diff
@@ -82,7 +82,8 @@ def from_xml_str(xml_str, handle=None):
     """
 
     try:
-        root_elem = ET.fromstring(xml_str)
+        xml_raw_str = ucsgenutils.add_escape_chars(xml_str)
```

Comment:
We should follow the model that is used for other invalid characters that are escaped. This may not be the right place to do it.